styleguide.config.js

```diff
@@ -101,7 +102,23 @@ module.exports = {
             base: '"Lato", sans-serif',
         },
     },
+    title: "Wonder Blocks",
     styles: {
+        StyleGuide: {
+            sidebar: {
+                width: "initial",
```

Comment:
Removes the "200px" limitation. Even just with "Wonder Blocks" as title, the sidebar wanted to be a touch wider to not wrap the text.